lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
That seems reasonable, but then the change should be to make the templates do the escaping, not to escape every string sent to the templates.

@muhammad-ammar Can you update the templates instead?